### PR TITLE
Create CV public page

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: templates/summary/_block/trainings.html.twig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            <h2>Formations</h2>
+            <p>
+                {% for training in summary.trainings %}
+                    <h3>{{ training.diploma }} - {{ training.studyField }}</h3>
+                    <h4>{{ training.organization|upper }}</h4>
+                    <span>{{ training.startedAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM Y')|capitalize }} - {{ training.endedAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM Y')|capitalize }} ( todo )</span>
+                    <br />
+                    <p>{{ training.description }}</p>
+                    <br />
+                    <i>{{ training.extracurricular }}</i>
```
Comment (HeahDude) [inline templates/summary/_block/trainings.html.twig]: Please use an upper "C".

File: templates/summary/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{% extends 'base.html.twig' %}
+
+{% block meta_noindex %}
```
Comment (HeahDude) [inline templates/summary/index.html.twig]: the block should be named `meta_index` IMO, what do you think Titouan?

File: templates/summary/_block/header.html.twig [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    {% endblock %}
+    {% block summary_right_part %}{% endblock %}
+    {% block summary_bottom_part %}
+        <div>
+            <div>
+                {{ summary.member.activatedAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM Y') }}<br>
+                Date d'adhésion
+            </div>
+            <div>
+                23 <br> participations à des events
```
Comment (alOneh) [inline templates/summary/_block/header.html.twig]: `23` should be a variable right ?

Comment (tgalopin) [inline templates/summary/_block/header.html.twig]: Indeed, but this is the topic of another ticket as there are multiple places in the page where we have to find past data about the user.

**Pull request merged**